stdlib/tkinter/__init__.pyi

```diff
@@ -800,7 +800,7 @@ class Tk(Misc, Wm):
     def wantobjects(self, *args, **kwargs): ...
     def willdispatch(self): ...
 
-def Tcl(screenName: str | None = None, baseName: str | None = None, className: str = "Tk", useTk: bool = ...) -> Tk: ...
+def Tcl(screenName: str | None = None, baseName: str | None = None, className: str = "Tk", useTk: bool = False) -> Tk: ...
```

Comment:
The `useTk` argument here is a bit weird. It never makes sense to set it to `True`, because you are supposed to instantiate `Tk()` directly.